draft-westerlund-tsvwg-sctp-crypto-chunk.md

```diff
@@ -549,25 +549,24 @@ Engine" extra cause code identifier 0x00.
 
 If the protection engine specifies a handshake for example for
 authentication, and key management is implemented inband, it may happen
-that the procedure has errors. In such case an ERROR chunk will be
+that the procedure has errors. In such case an ABORT chunk will be
```

Comment:
This is also a change from just an Error to an Abort. However, this change is likely correct as I don't see a path of continuing if the key-handshake failed.